src/task.py

```diff
@@ -30,6 +30,11 @@ async def check_announcements(context: ContextTypes.DEFAULT_TYPE) -> None:
             logger.exception(message)
             await context.bot.send_message(chat_id=LOGGER_CHAT_ID, text=message, disable_notification=True)
             continue
+        except ConnectionTimeoutError:
+            message = f"Connection Timeout while scraping {department.id}"
+            logger.exception(message)
+            await context.bot.send_message(chat_id=LOGGER_CHAT_ID, text=f"{message}\n\n{traceback.format_exc()}")
```

Comment:
No need to send the traceback here. That's the reason why we catch these specific errors actually, they are known, so simple message would do the job for us. If it's undefined, the broad catch block below will take care of it send us a traceback.